### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Jamulus/1-index](https://hosted.weblate.org/projects/jamulus/1-index/).

It also includes the following components:

* [Jamulus/QOS-Windows](https://hosted.weblate.org/projects/jamulus/qos-windows/)

* [Jamulus/Custom-Directories](https://hosted.weblate.org/projects/jamulus/custom-directories/)

* [Jamulus/Include-Client-Commands](https://hosted.weblate.org/projects/jamulus/include-client-commands/)

* [Jamulus/Include-Server-Commands](https://hosted.weblate.org/projects/jamulus/include-server-commands/)

* [Jamulus/Include-Shared-Commands](https://hosted.weblate.org/projects/jamulus/include-shared-commands/)

* [Jamulus/Server-Bandwidth](https://hosted.weblate.org/projects/jamulus/server-bandwidth/)

* [Jamulus/footertext](https://hosted.weblate.org/projects/jamulus/footertext/)

* [Jamulus/Contribution](https://hosted.weblate.org/projects/jamulus/contribution/)

* [Jamulus/Running-a-Server](https://hosted.weblate.org/projects/jamulus/running-a-server/)

* [Jamulus/Include-Backing-Up](https://hosted.weblate.org/projects/jamulus/include-backing-up/)

* [Jamulus/Privacy-Statement](https://hosted.weblate.org/projects/jamulus/privacy-statement/)

* [Jamulus/FAQ](https://hosted.weblate.org/projects/jamulus/faq/)

* [Jamulus/Software-Manual](https://hosted.weblate.org/projects/jamulus/software-manual/)

* [Jamulus/Tips-Tricks-More](https://hosted.weblate.org/projects/jamulus/tips-tricks-more/)

* [Jamulus/general](https://hosted.weblate.org/projects/jamulus/general/)

* [Jamulus/navigation](https://hosted.weblate.org/projects/jamulus/navigation/)

* [Jamulus/Installation-for-Windows](https://hosted.weblate.org/projects/jamulus/installation-for-windows/)

* [Jamulus/Installation-for-Linux](https://hosted.weblate.org/projects/jamulus/installation-for-linux/)

* [Jamulus/Installation-for-Macintosh](https://hosted.weblate.org/projects/jamulus/installation-for-macintosh/)

* [Jamulus/Client-Troubleshooting](https://hosted.weblate.org/projects/jamulus/client-troubleshooting/)

* [Jamulus/Command-Line-Options](https://hosted.weblate.org/projects/jamulus/command-line-options/)

* [Jamulus/Server-Troubleshooting](https://hosted.weblate.org/projects/jamulus/server-troubleshooting/)

* [Jamulus/Getting-Started](https://hosted.weblate.org/projects/jamulus/getting-started/)

* [Jamulus/Installation-for-Android](https://hosted.weblate.org/projects/jamulus/installation-for-android/)

* [Jamulus/Installation-for-iOS](https://hosted.weblate.org/projects/jamulus/installation-for-ios/)

Current translation status:

Weblate translation status

### CODE REVIEW

File: _translator-files/po/fr/Contribution.po [hunk 1]
```diff
@@ -59,10 +59,14 @@ msgstr "Si vous pensez qu'il manque de la documentation ou des informations ou q
...
 msgid "We use [Weblate](https://hosted.weblate.org/projects/jamulus/#languages) to translate our documentation. Read about our [documentation and translation process](https://github.com/jamulussoftware/jamuluswebsite/blob/release/README.md), and [introduce yourself](https://github.com/jamulussoftware/jamulus/discussions) if you’d like to get involved."
-msgstr "Si vous souhaitez aider à la traduction, vous devrez disposer d'un compte GitHub et être familier avec l'utilisation de git et de Jekyll. Lisez notre [processus de documentation et de traduction](https://github.com/jamulussoftware/jamuluswebsite/blob/release/README.md), et [présentez-vous](https://github.com/jamulussoftware/jamulus/discussions) si vous souhaitez vous impliquer. Vous pouvez vérifier l'état actuel des traductions du site web [ici](Statistics)."
+msgstr ""
+"Nous utilisons [Weblate](https://hosted.weblate.org/projects/jamulus/"
+"#languages) pour traduire notre documentation. Lire à propos notre ["
```
Comment (pljones) [inline _translator-files/po/fr/Contribution.po:65]: The line breaks here look prone to error.  Is it correct to insert a line break after a `[` or within a url?

Comment (ann0see) [reply _translator-files/po/fr/Contribution.po:65]: @trebmuh

Comment (trebmuh) [reply _translator-files/po/fr/Contribution.po:65]: As far as I know, it's not a problem at all. It's not really "cut" and will be "re-glued" by whatever is processing the file after.
I mean, at least in a lot of other software I'm (or have been) translating, this is the same.
Edit : oh, I forget to add that this is Weblate which is doing so (I mean, it's not chosen by the translator).

**Comment by ignotus666** [comment]
That long list of files after "It also includes the following components:" is because they all share the same settings as Index-1 and it was easier (and is supposed to use fewer resources) when setting up the project - and I wasn't aware that it would cause this. The PR message is a bit excessive and misleading, it's as if Index-1 were the "parent" file of the project and any changes affect it when they don't, and every other file always gets listed even if unchanged. Giving each doc its own settings can be done though it'd be quite tedious. The PR message can be changed to just "Translation update from Weblate" or something like that but tell me what you think first.

I've also enabled squashing commits before they get sent here. I've set them to be squashed by author but that can be changed to per language/file or everything into one commit. I think we won't see if it worked until this PR's merged and the next one is opened though.

**Comment by ignotus666** [comment]
Ok, squashing is working.

**Comment by ignotus666** [comment]
Could we merge this now? I'd like to remove obsolete .po files but if I do so before this is merged there might be conflicts (mainly with "Directories/"Custom-Directories").

**Comment by ignotus666** [comment]
Also @ann0see  - what do you think about simplifying the PR message?

**Comment by ann0see** [comment]
Yes sure. If the commit message contains the language and a short description, it should be ok.

**Comment by ann0see** [comment]
> Could we merge this now?

Yes

**Comment by ignotus666** [comment]
> Yes sure. If the commit message contains the language and a short description, it should be ok.

The problem with this is that more languages can be added to a PR after it's initially submitted, which could make it misleading (probably why it doesn't include languages in the PR message now either).

I was thinking of just "Translations update from Hosted Weblate" in the title and maybe add "Check commits for languages" to the body (there's 1 commit for each language). And remove all those references to the entire list of files.

**Comment by ann0see** [comment]
I'd remove the "Hosted" part. 


Probably @hoffie has better insight in good commit messages as he's the person who introduced some style in the main repo.

**Comment by ann0see** [comment]
@ignotus666 you need to translate directoryfile and listfilter probably again. after I resolved the conflicts

**Comment by ann0see** [comment]
I think fixing conflicts on the weblate repo is more difficult. I now did it on GitHub the standard way.

**Pull request merged**